### PR TITLE
i18n(es): Update of `api-reference.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description. 
Update of `api-reference.mdx` based on this commit f796771a17e903e65925f1476e72a47c8ed01e2d

### CODE REVIEW

File: src/pages/es/reference/api-reference.mdx [hunk 1]
```diff
@@ -731,13 +728,9 @@ Esta función acepta las siguientes propiedades:
 
 **Type:** `TSchema extends ZodType`
 
-`schema` es un objeto Zod opcional para configurar el tipo y la forma del frontmatter del documento para una colección. Cada valor de objeto debe usar [un validador Zod](https://github.com/colinhacks/zod).
+El`schema` es un objeto de Zod opcional para configurar el tipo y la forma del frontmatter del documento para una colección. Cada valor debe usar [un validador de Zod](https://github.com/colinhacks/zod).
```
Comment (kevinzunigacuellar) [inline src/pages/es/reference/api-reference.mdx:731]:
```suggestion
El `schema` es un objeto de Zod opcional para configurar el tipo y la forma del frontmatter del documento para una colección. Cada valor debe usar [un validador de Zod](https://github.com/colinhacks/zod).
```

**Comment by kevinzunigacuellar** [comment]
LGTM!

**Comment by dreyfus92** [comment]
Gracias por el review @kevinzunigacuellar 💚

**Review by delucis — APPROVED** [review]
Thank you for updating this @dreyfus92 and for reviewing @kevinzunigacuellar! 💜

**Pull request merged**